.github/workflows/tests.yml

```
@@ -32,8 +32,8 @@ jobs:
       - name: Set Windows environment
         if: matrix.os == 'windows-latest'
         run:
```

Looks like an existing bug in the workflow, but this should be:

```suggestion
        run: |
```
to preserve the line break between the two subsequent `echo ...` lines.